UI/DropsController.gd

```diff
@@ -108,6 +108,32 @@ func _on_DropsTableItemsPanel_dropped_item(drop_item: GatheringItem):
 	_selected_drops_table.add_item(drop_item)
 	print_debug("Placed %s in drops table %s" % [tr(drop_item.item.name), _selected_drops_table.name])
 
+func _on_DropsFilterLineEdit_text_changed(new_text):
+	_FilterList(new_text)
+	
+func _FilterList(filter_text: String = ""):
```

Comment:
Instead of a new function, this could also be part of `_update_drops_tables()` and do the filtering in there, but i'll leave that choice up to your liking